lib/matplotlib/axes/_base.py

```diff
@@ -1363,11 +1363,9 @@ def set_anchor(self, anchor, share=False):
         anchor defines where the drawing area will be located within the
         available space.
 
-        .. ACCEPTS: [ 'C' | 'SW' | 'S' | 'SE' | 'E' | 'NE' | 'N' | 'NW' | 'W' ]
-
         Parameters
         ----------
-        anchor : str or 2-tuple of floats
+        anchor : 2-tuple of floats or {'C', 'SW', 'S', 'SE', ...}
```

Comment:
Is the ellipsis because its obvious what the other arguments are?  Maybe not so obvious for non-english readers?

Reply:
It’s explained in the following docstring.